xc7/utils/prjxray_routing_import.py

```diff
@@ -228,6 +229,14 @@ def check_feature(feature):
     if rebuf_key in REBUF_SOURCES:
         return ' '.join([feature] + REBUF_NODES[REBUF_SOURCES[rebuf_key]])
 
+    m = IOI_OCLK.fullmatch(feature_path[1])
```

Comment:
I think this is better modelled as a change in the techmap, because I believe this routing is a function of the mode?

Reply:
Ok, I will try to implement as suggested. OCLKM should be directed to the CLKB port, so actually it should be possible to add that to the pb_type as well and to the techmap.

Reply:
By trying to add the CLKB port as well, I get a fasm lookup error. The CLK signal is taken from the OCLK_0 port instead of the OCLKM_0. 

```
prjxray.fasm_assembler.FasmLookupError: Segment DB LIOI3, key IOI3.IOI_OLOGIC0_CLKB.IOI_OCLK_0 not found from line IOI3_X0Y25.IOI_OLOGIC0_CLKB.IOI_OCLK_0'
```

Being OCLKM a pip, I am unsure on how to do this in the techmap.
Moreover, OCLKM is dedicated to only route the clock signal that goes through the inverter, both for ILOGIC and OLOGIC. I do believe, that, whenever a CLK is used in the I/OLOGIC, the same signal needs to reach the inverted CLK port which will give "stability" to the primitive. In fact, I have seen that, without the OCLKM pip enabled, the OSERDES was alive, but was highly unstable. IMO it is safe to allow `check_features` to always add the OCLKM pip feature anytime the OCLK is enabled.

Reply:
Sounds good to me.  We can always change it later.